test/parallel/test-benchmark-streams.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+require('../common');
+
+const runBenchmark = require('../common/benchmark');
+
+runBenchmark('streams',
+             [
+               'kind=duplex',
```

Comment:
So what we've done in the past for this kind of thing is rather than add multiple `kind=` statements, modify the benchmarks to provide their own default value if `kind` is empty.  See, for example, `type` and `method` in `test-benchmark-misc.js`.